example/app/(tabs)/index.tsx

```diff
@@ -97,16 +103,28 @@ export default function TabOneScreen() {
           candleClient
             .getLinkedAccounts()
             .then((accounts) => {
-              if (accounts.length > 0) {
-                console.log("Linked accounts:", accounts[0].details);
-              } else {
-                console.log("Linked accounts:", accounts);
-              }
+              accounts.forEach((account) => {
+                switch (account.state) {
+                  case "active":
+                    console.log(
+                      `${account.service} is active`,
+                      JSON.stringify(account, null, 2)
```

Comment:
#req JSON.stringify is dangerous (it only works if every field recursively is a primitive, but doesn't enforce that at the type level). Also the output is much uglier than `util.inspect`.

```suggestion
                      util.inspect(account, { depth: null, colors: true, compact: false })
```

Reply:
I've never had a problem with JSON.stringify. This is only the example app too.

Reply:
It's still dangerous whether you've experienced it throwing or not (I have). Do you see any advantage to using `JSON.stringify` over `util.inspect`?

Reply:
It's a node function, not built into RN.